app/Contact.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Contact extends Model
+{
+    protected $fillable = [
+        'first_name',
+        'last_name',
+        'email',
+        'city',
+        'country',
+        'job_title'
+    ];
```

Comment:
[Good Thing]This is a good practice to follow since we have decided to do a mass-assigning of few fields in this model which help to prevent again SQL Injection.